feature/web support aspect-ratio 16/9 in older versions

JeisonVargasGlobant opened the pull request:
Add validation to handle aspect-ratio 16/9 with padding-top: 56.25% to support older versions

tvanlaerhoven commented on src/internal/THEOplayerView.web.tsx:
```diff
@@ -53,11 +53,19 @@ export function THEOplayerView(props: React.PropsWithChildren<THEOplayerViewProp
+        // Handle aspect-ratio 16/9 with padding-top: 56.25% to support older versions.
+        // {@link https://www.w3schools.com/howto/howto_css_aspect_ratio.asp}
+        <div style={styles.containerNoAspectRatio}>
+          <div ref={container} style={styles.absoluteFillNoAspectRatio} className="theoplayer-container" />
```
I added a `className` here.

JeisonVargasGlobant commented:
> Many thanks @JeisonVargasGlobant ! I restructured the PR a bit: extracted the styles, added some comments and a changelog.
> 
> The only thing I added was the `className` on the container. It's also there in the regular flow and allows for overriding/modifying the container from the outside. Some customers use this. Maybe check if this is OK for you like this?

It's OK. Thanks
